server/package.json

```diff
@@ -16,7 +16,7 @@
         "express": "~4.16.1",
         "http-errors": "~1.6.3",
         "jest": "^26.6.3",
-        "mongoose": "^5.11.17",
+        "mongoose": "^5.11.15",
```

Comment:
why the downgrade?

Reply:
just to get rid of deprecation warnings 😅